pkg/rm/tcc/tcc_service_test.go

```diff
@@ -0,0 +1,33 @@
+package tcc
+
+import (
+	"sync"
+	"testing"
+)
+
+func TestTCCServiceProxy_RegisterResource(t1 *testing.T) {
+	type fields struct {
+		referenceName        string
+		registerResourceOnce sync.Once
+		TCCResource          *TCCResource
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		wantErr bool
+	}{}
```

Comment:
There seems to be no test data defined here

Reply:
I will add it